src/config/feature-flags.ts

```diff
@@ -21,6 +21,7 @@ export enum BooleanFlags {
 }
 
 export enum StringFlags {
+	GITHUB_SCOPES = "github-scopes",
```

Comment:
The Feature Flag is not ready yet. Having permission issues while creating in LaunchDarkly..
Working on it!